src/Stratis.Bitcoin/Builder/FullNodeFeatureExecutor.cs

```diff
@@ -103,21 +86,13 @@ private void Execute(Action<IFullNodeFeature> callback, bool reverseOrder = fals
                     }
                     catch (Exception ex)
                     {
-                        if (exceptions == null)
-                        {
-                            exceptions = new List<Exception>();
-                        }
-
-                        exceptions.Add(ex);
+                        this.logger.LogError("Exception occurred: {0}", ex.ToString());
```

Comment:
I agree that for intialization, we don't want to continue.
For disposing of resources, however, I'd be quite keen for the node to dispose of as many resources as possible, rather than bomb out on the first problem and then not finishing the others gracefully.

Reply:
We are not supposed to throw from initialize or dispose methods. 
I think this is something that should be for the component to decide. For example if we expect some kind of error in initialization or disposal and it's not critical (we will be able to run node properly after initialize failed or run it next time when dispose fails)- then the components itself should handle this exception and swallow it with some kind of error message to the console. 

Before we used to catch all exceptions which didn't seem correct. To me this was more like a sanity check- that kind of stuff should never happen.

Reply:
> We are not supposed to throw from initialize or dispose methods.
but if we do throw in Dispose, and we do, you'll exit at the first problem and won't try to dispose the other features.

>  the components itself should handle this exception and swallow it with some kind of error message to the console.
but we don't have that yet.